src/components/LocationBias.tsx

```diff
@@ -11,17 +11,18 @@ import LoadingIndicator from '../icons/LoadingIndicator';
  * @public
  */
 export interface LocationBiasCssClasses {
-  container?: string,
+  locationBiasContainer?: string,
   location?: string,
   source?: string,
   button?: string,
   loadingIndicatorContainer?: string
 }
 
 const builtInCssClasses: Readonly<LocationBiasCssClasses> = {
-  container: 'text-sm text-neutral text-center flex justify-center items-center',
-  location: 'font-semibold mr-1',
-  button: 'text-primary hover:underline focus:underline ml-1',
+  locationBiasContainer: 'text-sm text-neutral text-center justify-center items-center flex flex-col lg:flex-row',
+  location: 'font-semibold mr-1 lg:ml-7',
+  source: 'ml-3 lg:ml-0 ',
```

Comment:
why is the `ml-3` needed? also there's an extra space at the end

Reply:
Since we want to remove the dash on mobile, I made a new `span` that holds the dash.  It becomes invisible on smaller screens.  But, invisible objects still take space, so I had to add some padding to the right of the `source` to keep the attribution message centered.  It's the same solution I used to keep the location button centered even with the loading indicator.